core/frontend/src/DisplayStyleState.ts

```diff
@@ -395,22 +397,30 @@ export abstract class DisplayStyleState extends ElementState implements DisplayS
     return (index < 0 || index >= layers.length) ? undefined : layers[index];
   }
 
-  /** @internal */
-  public changeBaseMapProps(props: MapLayerProps | ColorDef) {
+  /** @beta */
+  public changeBaseMapProps(props: ColorDef | MapLayerProps | BackgroundMapProviderProps) {
     if (props instanceof ColorDef) {
-      const transparency = this.settings.mapImagery.backgroundBase instanceof ColorDef ? this.settings.mapImagery.backgroundBase.getTransparency() : 0;
+      let transparency = 0;
+      if (this.settings.mapImagery.backgroundBase instanceof ColorDef)
+        transparency = this.settings.mapImagery.backgroundBase.getTransparency();
       this.settings.mapImagery.backgroundBase = props.withTransparency(transparency);
+    } else if (BackgroundMapProvider.isMatchingProps(props)) {
+      // props is BackgroundMapProviderProps
+      if (this.settings.mapImagery.backgroundBase instanceof BackgroundMapProvider) {
+        const mapProvider = this.settings.mapImagery.backgroundBase;
+        this.settings.mapImagery.backgroundBase = mapProvider.clone(props as BackgroundMapProviderProps);
+      } else {
+        this.settings.mapImagery.backgroundBase = BackgroundMapProvider.fromJSON(props as BackgroundMapProviderProps);
+      }
     } else {
+      // props is MapLayerProps
```

Comment:
Not necessarily - `{ }` is a valid BackgroundMapProviderProps; in this context it means "don't change anything".